.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: Test
```

Comment:
Thinking about naming here - do we think it valuable to have a more specific title than `test`? Maybe `lint_geosupport_unit_tests`?

Reply:
the individual jobs in the workflow actually do have the more specific names. so they show up like this:
<img width="519" alt="image" src="https://user-images.githubusercontent.com/7444289/223204137-faa26f5c-6015-41a2-8a12-e1b9f7e77150.png">